### PR TITLE
Redefine a Kotlin configuration

### PR DESCRIPTION
**Pull request by 6r1d**
Updates the Kotlin configuration according to Bogdan's suggestion.
We're not using the private repos this way.

Signed-off-by: 6r1d <vic.6r1d@gmail.com>

### CODE REVIEW

File: src/guide/kotlin-java.md [hunk 1]
```diff
@@ -14,27 +14,33 @@ that you know how to build and deploy your program on the target platforms.
...
+    implementation(platform("org.jetbrains.kotlin:kotlin-bom"))
+    // Use the Kotlin JDK 8 standard library.
+    implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8")
+    // This dependency is used by the application.
+    implementation("com.google.guava:guava:31.0.1-jre")
+    // Use the Kotlin test library.
+    testImplementation("org.jetbrains.kotlin:kotlin-test")
+    // Use the Kotlin JUnit integration.
+    testImplementation("org.jetbrains.kotlin:kotlin-test-junit")
+    // Iroha-related
```
Comment (outoftardis) [inline src/guide/kotlin-java.md:39]:
```suggestion
    // Load Iroha-related dependencies
```

File: src/guide/kotlin-java.md [hunk 2]
```diff
@@ -14,27 +14,33 @@ that you know how to build and deploy your program on the target platforms.
...
-    implementation "jp.co.soramitsu.iroha2-java:testcontainers:iroha2-dev-SNAPSHOT"
+    // Align versions of all Kotlin components
+    implementation(platform("org.jetbrains.kotlin:kotlin-bom"))
+    // Use the Kotlin JDK 8 standard library.
+    implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8")
+    // This dependency is used by the application.
+    implementation("com.google.guava:guava:31.0.1-jre")
+    // Use the Kotlin test library.
+    testImplementation("org.jetbrains.kotlin:kotlin-test")
+    // Use the Kotlin JUnit integration.
```
Comment (outoftardis) [inline src/guide/kotlin-java.md:37]:
```suggestion
    // Use the Kotlin test library
    testImplementation("org.jetbrains.kotlin:kotlin-test")
    // Use the Kotlin JUnit integration
```

File: src/guide/kotlin-java.md [hunk 3]
```diff
@@ -14,27 +14,33 @@ that you know how to build and deploy your program on the target platforms.
...
 
 dependencies {
-    implementation "jp.co.soramitsu.iroha2-java:client:iroha2-dev-SNAPSHOT"
-    implementation "jp.co.soramitsu.iroha2-java:block:iroha2-dev-SNAPSHOT"
-    implementation "jp.co.soramitsu.iroha2-java:model:iroha2-dev-SNAPSHOT"
-    implementation "jp.co.soramitsu.iroha2-java:test-tools:iroha2-dev-SNAPSHOT"
-    implementation "jp.co.soramitsu.iroha2-java:testcontainers:iroha2-dev-SNAPSHOT"
+    // Align versions of all Kotlin components
+    implementation(platform("org.jetbrains.kotlin:kotlin-bom"))
+    // Use the Kotlin JDK 8 standard library.
```
Comment (outoftardis) [inline src/guide/kotlin-java.md:31]:
```suggestion
    // Use the Kotlin JDK 8 standard library
```

File: src/guide/kotlin-java.md [hunk 4]
```diff
@@ -14,27 +14,33 @@ that you know how to build and deploy your program on the target platforms.
...
+    implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8")
+    // This dependency is used by the application.
```
Comment (outoftardis) [inline src/guide/kotlin-java.md:33]: I'd rephrase this similarly to other comments to start with the imperative voice

**Review by outoftardis — APPROVED** [review]

File: src/guide/kotlin-java.md (hunk 4 above)
Comment (appetrosyan) [reply src/guide/kotlin-java.md:33]: This should probably apply to the code in the SDK
